.infrastructure/deployment.yml

```diff
@@ -18,37 +18,56 @@ spec:
         app: todoapp
     spec:
       containers:
-      - name: todoapp
-        image: ikulyk404/todoapp:3.0.0
-        resources:
-          requests:
-            memory: "256Mi"
-            cpu: "150m"
-          limits:
-            memory: "256Mi"
-            cpu: "150m"
-        env:
-        - name: PYTHONUNBUFFERED
-          valueFrom:
-            configMapKeyRef:
-              name: app-config
-              key: PYTHONUNBUFFERED
-        - name: SECRET_KEY
-          valueFrom:
-            secretKeyRef:
-              name: app-secret
-              key: SECRET_KEY
-        ports:
-        - containerPort: 8080
-        livenessProbe:
-          httpGet:
-            path: api/health
-            port: 8080
-          initialDelaySeconds: 60
-          periodSeconds: 5
-        readinessProbe:
-          httpGet:
-            path: api/ready
-            port: 8080
-          initialDelaySeconds: 5
-          periodSeconds: 5
+        - name: todoapp
+          image: ikulyk404/todoapp:3.0.0
+          volumeMounts:
+            - name: data
+              mountPath: /app/data
+            - name: app-secrets-volume
+              mountPath: /app/secrets
+              readOnly: true
+            - name: app-config-volume
+              mountPath: /app/configs
+              readOnly: true
+          resources:
+            requests:
+              memory: "256Mi"
+              cpu: "150m"
+            limits:
+              memory: "256Mi"
+              cpu: "150m"
+          env:
+            - name: PYTHONUNBUFFERED
+              valueFrom:
+                configMapKeyRef:
+                  name: app-config
+                  key: PYTHONUNBUFFERED
+            - name: SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: app-secret
+                  key: SECRET_KEY
+          ports:
+            - containerPort: 8080
+          livenessProbe:
+            httpGet:
+              path: api/health
+              port: 8080
+            initialDelaySeconds: 60
+            periodSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: api/ready
```

Comment:
The path for the readiness probe is specified as `api/ready`. Similar to the liveness probe, ensure that this path is correct and accessible within the application. It might need a leading slash, like `/api/ready`, depending on how the application routes are set up.